Rtp/RtpClient.cs

```diff
@@ -2291,7 +2291,8 @@ public void DisconnectSockets()
                     MulticastGroups.Clear();
 
                     //For Udp the RtcpSocket may be the same socket as the RtpSocket if the sender/reciever is duplexing
-                    if (RtcpSocket is not null && RtpSocket.Handle.Equals(RtcpSocket.Handle) is false) RtcpSocket.Close();
+                    if (RtcpSocket is not null && RtcpSocket.Handle.Equals(RtpSocket?.Handle) is false)
```

Comment:
RtcpSocket.IsNullOrDisposed() is false, because both sockets could be disposed and this check would pass